sympy/printing/pycode.py

```diff
@@ -296,6 +296,14 @@ def _print_Sum(self, expr):
     def _print_ImaginaryUnit(self, expr):
         return '1j'
 
+    def _print_KroneckerDelta(self, expr):
+        a,b = expr.args
```

Comment:
Normal convention is to have a space like `a, b = expr.args`

Reply:
Fixed.